usage-data.adoc

```diff
@@ -0,0 +1,25 @@
+= Usage data
+
+Overview of the data CodeReady Containers collect provided the user consented (see README).
```

Comment:
"The following table describes the opt-in usage data collected by CodeReady Containers."